lib/SPIRV/libSPIRV/SPIRVEntry.h

```diff
@@ -950,6 +950,21 @@ template <> struct InstToContinued<OpSpecConstantComposite> {
   constexpr static spv::Op OpCode = OpSpecConstantCompositeContinuedINTEL;
 };
 
+class SPIRVModuleProcessed : public SPIRVEntryNoId<OpModuleProcessed> {
```

Comment:
The spec says this instruction was missing before SPIRV 1.1. Should we implement `getRequiredSPIRVVersion` then?